liteclient/connection.go

```diff
@@ -38,6 +40,50 @@ type connection struct {
 	pool *ConnectionPool
 }
 
+func (c *ConnectionPool) AddConnetionsFromConfig(ctx context.Context, config GlobalConfig) error {
+	g, ctx := errgroup.WithContext(ctx)
+	success := 0
+
+	for _, ls := range config.Liteservers {
+		ip := intToIP4(ls.IP)
+		conStr := fmt.Sprintf("%s:%d", ip, ls.Port)
+		ls := ls
+
+		g.Go(func() error {
+			err := c.AddConnection(ctx, conStr, ls.ID.Key)
+			if err == nil {
+				success++
```

Comment:
concurrent access to `success`, we can use sync.AddInt32 here, to not have a race